libs/blocks/global-navigation/global-navigation.css

```diff
@@ -76,6 +76,11 @@
   background-color: var(--feds-background-nav--light);
 }
 
+[dir = "rtl"] .feds-nav-wrapper {
```

Comment:
nitpick - there's mixed `dir='rtl'` and `dir = "rtl"` throughout the code, we should be consistent.